docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -76,7 +76,10 @@ include::{sourcedir}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[tag
 
 <1> If you wish to use off-heap, you'll have to define resource pool, giving the memory size you want to allocate.
 
-Do not forget to define in the java options the -XX:MaxDirectMemorySize option, according to the off-heap size you intend to use.
+The example above allocates a very small amount of off-heap.
+Remember that data stored off-heap will have to be serialized and deserialized.
+You should thus favor off-heap for large amounts of data where storing them on heap would have too severe impacts on garbage collection.
+Do not forget to define in the java options the `-XX:MaxDirectMemorySize` option, according to the off-heap size you intend to use.
```

Comment:
This seems to imply setting these to the same value... thats not really what we want... but I'm not sure what we should say instead.